src/display/canvas.js

```diff
@@ -1589,21 +1599,37 @@ class CanvasGraphics {
   }
 
   save() {
-    this.ctx.save();
+    if (this.inSMaskMode()) {
+      // SMask mode may be turned on/off causing us to loose graphics state.
```

Comment:
Grammar-nit: I believe that you probably want to change `to loose` -> `to lose`.